src/client/visualizations/line-chart-old/line-chart.scss

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2015-2016 Imply Data, Inc.
+ * Copyright 2017-2019 Allegro.pl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '../../imports';
+
+.line-chart {
+  .measure-line-charts {
+    @include internet-explorer {
+      overflow-y: hidden;
+    }
+    overflow-x: hidden;
+    overflow-y: auto;
+    padding-left: $vis-h-padding;
+  }
+
+  .measure-line-chart {
```

Comment:
It's copied from old line chart. Anything below this line is not used and should be placed in specific component's styles.